Doc/c-api/intro.rst

```diff
@@ -90,9 +94,11 @@ multi-platform builds since the platform independent headers under
 :envvar:`prefix` include the platform specific headers from
 :envvar:`exec_prefix`.
 
-C++ users should note that though the API is defined entirely using C, the
-header files do properly declare the entry points to be ``extern "C"``, so there
-is no need to do anything special to use the API from C++.
+.. note::
```

Comment:
I don't think this needs its own block.

Reply:
I was basing this change on the [devguide](https://devguide.python.org/documenting/#paragraph-level-markup)'s summary for using the ``note`` directive:

> An especially important bit of information about an API that a user should be aware of when using whatever bit of API the note pertains to. The content of the directive should be written in complete sentences and include all appropriate punctuation.

This section seemed to fit that description, but I can remove it if you disagree.

Reply:
A better candidate for the ``note`` directive though might be the area where I was previously was using ``important``.

Reply:
Obviously, this is subjective. But I don't think information about C++ compatibility is any more important than anything else in this section.

Generally, we want to avoid turning the docs in a a series of multi-colored boxes.